### ISSUE TITLE
After training, load the .pt file for inference audio (incoherent speech).

### ISSUE THREAD

**Issue by 11075225** [SWivid/F5-TTS#1231]
### Checks

- [x] This template is only for research question, not usage problems, feature requests or bug reports.
- [x] I have thoroughly reviewed the project documentation and read the related paper(s).
- [x] I have searched for existing issues, including closed ones, no similar questions.
- [x] I am using English to submit this issue to facilitate community communication.

### Question details

[audio (3).wav](https://github.com/user-attachments/files/23843542/audio.3.wav)
The model was fine-tuned using the Emilia_Malaysian dataset, and the audio effects generated during inference testing using the .pt files produced during training are shown above.Here are my training instructions:
accelerate launch --mixed_precision=fp16 src/f5_tts/train/train.py \
--config-name F5TTS_v1_Base.yaml \
++optim.num_warmup_updates=5000 \
++optim.learning_rate=2.5e-5 \
++datasets.name=Emilia_Malaysian \
++datasets.batch_size_per_gpu=8192 \
++model.arch.checkpoint_activations=True \
++ckpts.save_per_updates=10000 \
++ckpts.last_per_updates=5000

**Comment by 11075225** [comment]
I can generate speech normally using the first 5000 .pt files, but this problem occurs when using the later .pt files. I am training based on the official model_1250000.safetensors.

**Comment by SWivid** [comment]
>  fine-tuned
> 
> accelerate launch --mixed_precision=fp16 src/f5_tts/train/train.py

1. have you made a copy of pretrained ckpt file under corresponding path (if you don't know what i am taking about, then you are not doing finetuning)
2. use_ema is true or false?

> 5000 .pt 
> ++optim.num_warmup_updates=5000

i am not sure if you have trained well the model.

> [audio (3).wav](https://github.com/user-attachments/files/23843542/audio.3.wav)

what is the ref_audio ref_text gen_text, it sounds like english.

> This template is only for research question, not usage problems, feature requests or bug reports.

please use correct template and provide necessary information.

**Comment by 11075225** [comment]
> > fine-tuned
> > accelerate launch --mixed_precision=fp16 src/f5_tts/train/train.py
> 
> 1. have you made a copy of pretrained ckpt file under corresponding path (if you don't know what i am taking about, then you are not doing finetuning)
> 2. use_ema is true or false?
> 
> > 5000 .pt
> > ++optim.num_warmup_updates=5000
> 
> i am not sure if you have trained well the model.
> 
> > [audio (3).wav](https://github.com/user-attachments/files/23843542/audio.3.wav)
> 
> what is the ref_audio ref_text gen_text, it sounds like english.
> 
> > This template is only for research question, not usage problems, feature requests or bug reports.
> 
> please use correct template and provide necessary information.

1. I am training based on the official model_1250000.safetensors, copy pretrained ckpt file under corresponding path.
2. use_ema is the default value; I haven't changed it.
3. I used English audio to do voice cloning to generate Malay audio.

**Comment by SWivid** [comment]
> what is the ref_audio ref_text gen_text, it sounds like english.

need to provide these to analyse

would suggest train longer (100k 200k would be good to see) cuz you have just finish the warmup; and turn use_ema to false to test at early stage of training as mentioned in readme.

**Comment by 11075225** [comment]
> > what is the ref_audio ref_text gen_text, it sounds like english.
> 
> need to provide these to analyse
> 
> would suggest train longer (100k 200k would be good to see) cuz you have just finish the warmup; and turn use_ema to false to test at early stage of training as mentioned in readme.

Below is my training curve chart. I used a .pt file that went through 200,000 iterations for inference, and the English audio I generated is also gibberish, not just Malay.
<img width="439" height="741" alt="Image" src="https://github.com/user-attachments/assets/3434b3bb-24ce-4f2d-a3fd-ac5e9d32fcb5" />

**Comment by VladCiocan** [comment]
Hi, I have a lot of issues trying to train this model, could you give me the list of versions used. 
I tried using :
=== SYSTEM ===
OS: Linux-5.14.0-570.26.1.el9_6.x86_64-x86_64-with-glibc2.34
Kernel: 5.14.0-570.26.1.el9_6.x86_64
Python: 3.11.14 (main, Oct 21 2025, 18:31:21) [GCC 11.2.0]
Executable: /home/user/miniconda3/envs/f5-tts/bin/python

=== CUDA / NVIDIA ===
CUDA_VISIBLE_DEVICES: 0
nvidia-smi:
Thu Jan  8 21:59:20 2026
+-----------------------------------------------------------------------------------------+
| NVIDIA-SMI 575.57.08              Driver Version: 575.57.08      CUDA Version: 12.9     |
|-----------------------------------------+------------------------+----------------------+
| GPU  Name                 Persistence-M | Bus-Id          Disp.A | Volatile Uncorr. ECC |
| Fan  Temp   Perf          Pwr:Usage/Cap |           Memory-Usage | GPU-Util  Compute M. |
|                                         |                        |               MIG M. |
|=========================================+========================+======================|
|   0  NVIDIA H200 NVL                Off |   00000000:06:10.0 Off |                    0 |
| N/A   25C    P0             93W /  600W |   28227MiB / 143771MiB |      0%      Default |
|                                         |                        |             Disabled |
+-----------------------------------------+------------------------+----------------------+
|   1  NVIDIA H200 NVL                Off |   00000000:06:11.0 Off |                    0 |
| N/A   24C    P0             90W /  600W |  134086MiB / 143771MiB |      0%      Default |
|                                         |                        |             Disabled |
+-----------------------------------------+------------------------+----------------------+


nvcc:
nvcc: NVIDIA (R) Cuda compiler driver
Copyright (c) 2005-2025 NVIDIA Corporation
Built on Tue_May_27_02:21:03_PDT_2025
Cuda compilation tools, release 12.9, V12.9.86
Build cuda_12.9.r12.9/compiler.36037853_0

=== PYTORCH ===
torch: 2.8.0+cu129
torch cuda: 12.9
cuda available: True
device count: 1
  GPU 0: NVIDIA H200 NVL
cudnn: 91002
bf16: True
SDPA flash: True
SDPA mem_efficient: True
SDPA math: True

=== TORCH PACKAGES ===
torchaudio: 2.8.0+cu129
torchvision: 0.23.0+cu129
